src/alfasim_score/converter/alfacase/score_input_reader.py

```diff
@@ -97,3 +140,96 @@ def read_lithology_materials(self) -> List[Dict[str, Union[Scalar, str]]]:
                 }
             )
         return lithology_data
+
+    def read_packer_fluid(self) -> List[Dict[str, Union[Scalar, str]]]:
+        """ "Get the properties of fluid above packer."""
+        # TODO PWPA-1970: review this fluid default with fluid actually used by SCORE file
+        # the fluid used now is water
+        return [
+            {
+                "name": FLUID_DEFAULT_NAME,
+                "type": "fluid",
+                "density": Scalar(1000.0, "kg/m3", "density"),
+                "thermal_conductivity": Scalar(0.6, THERMAL_CONDUCTIVITY_UNIT),
+                "specific_heat": Scalar(4181.0, SPECIFIC_HEAT_UNIT),
+                "thermal_expansion": Scalar(0.0004, THERMAL_EXPANSION_UNIT),
+            }
+        ]
+
+    def read_casings(self) -> List[Dict[str, Any]]:
+        """Read the data for the casing from SCORE input file"""
+        casing_data = []
+        for item in self.input_content["operation"]["thermal_simulation"]["well_strings"]:
```

Comment:
I remember João Paulo telling us that `operation/thermal_simulation` was the pysafe ouptut. Isn't it correct?
If so, we need to get `well_strings` from root.

Reply:
Yes, I think he said this about pysafe, I took note specifically about `tubing_string`
> Indeed there are a lot of duplicated fields, e.g. tubing_strings. Especially prioritize the one in the operation because they can be different from the one in root of json
I think that's better make this we suggested.

Reply:
done